axon-explorer/Makefile

```diff
@@ -0,0 +1,2 @@
+deploy: 
+	ansible-playbook start.yml
```

Comment:
Missing newline at the end of the file.

Reply:
At a minimum, a stop script and a data cleanup script are required。